### ISSUE TITLE
missing_docs rule broken when using Xcode8 GM

### ISSUE THREAD

**Issue by mgrebenets** [realm/SwiftLint#811]
After upgrading to Xcode8 GM the `missing_docs` rule is wrongly reported by `swiftlint`.

With Xcode 7.3.1 there are 0 violations.

```
DEVELOPER_DIR=/Applications/Xcode7.3.1.app/Contents/Developer swiftlint

# Done linting! Found 0 violations, 0 serious in 7 files.
```

With Xcode 8 all properly documented public functions and properties are reported as errors:

```
DEVELOPER_DIR=/Applications/Xcode8.app/Contents/Developer swiftlint

# Done linting! Found 23 violations, 0 serious in 7 files.
```

The example of properly documented code which is wrongly reported as incorrect:

```
    /// Unique identifier
    ///
    /// Used to create unique directory to store manager's data
    public let identifier: String
```

The documentation comment is properly rendered by Xcode, so it's correct (I assume).

![image](https://cloud.githubusercontent.com/assets/350654/18372969/b11c54b0-7683-11e6-9a00-7125847b1c7c.png)

**Comment by mgrebenets** [comment]
Apologies if I appear to bothering, but are there any news on this one?
I see a couple of related PRs, but latest SwiftLint (`0.12.0`) still has this issue.

Is this something that may be fixed in next release?
Or do we have to migrate our projects to Swift 3 maybe?

**Comment by norio-nomura** [comment]
As far as we know, false positive of `missing_docs` happens on Swift3.  https://github.com/realm/SwiftLint/issues/728
Are you experiencing on Swift 2.3?

**Comment by Loyolny** [comment]
I have this issue on Swift 2.3

**Comment by norio-nomura** [comment]
Ah, so sorry, I forgot that `missing_docs` rule has false positive issues before Swift 3 comes. 🙇

**Comment by jpsim** [comment]
This is a duplicate of #728. Please see my comment there (https://github.com/realm/SwiftLint/issues/728#issuecomment-263009386) for more details and suggestions on how to resolve the situation. Thanks!
